tools/SourceKit/lib/SwiftLang/SwiftASTManager.cpp

```diff
@@ -427,11 +441,11 @@ resolveSymbolicLinksInInputs(FrontendInputsAndOutputs &inputsAndOutputs,
   return replacementInputsAndOutputs;
 }
 
-bool SwiftASTManager::initCompilerInvocation(CompilerInvocation &Invocation,
-                                             ArrayRef<const char *> OrigArgs,
-                                             DiagnosticEngine &Diags,
-                                             StringRef UnresolvedPrimaryFile,
-                                             std::string &Error) {
+bool SwiftASTManager::initCompilerInvocation(
+    CompilerInvocation &Invocation, ArrayRef<const char *> OrigArgs,
+    DiagnosticEngine &Diags, StringRef UnresolvedPrimaryFile,
+    std::string &Error,
+    llvm::IntrusiveRefCntPtr<llvm::vfs::FileSystem> FileSystem) {
```

Comment:
I'm not sure what is the idiomatic way of adding new parameters to a function signature that has output parameters. I've no opinion on this, but wanted to flag this and similar functions that take `&Error` for potential discussion.

Reply:
will be addressed by the small refactoring